nucleus/src/main/java/za/co/absa/subatomic/infrastructure/team/view/jpa/MembershipRequestEntity.java

```diff
@@ -0,0 +1,34 @@
+package za.co.absa.subatomic.infrastructure.team.view.jpa;
+
+import lombok.*;
```

Comment:
Avoid wildcard imports. When our code quality stuff is added this will not be allowed.
Set IntelliJ to not do this by:

<img width="1095" alt="screen shot 2018-02-15 at 15 25 39" src="https://user-images.githubusercontent.com/1175891/36258746-8971694a-1264-11e8-813b-581252ce6ec5.png">

Reply:
I.e. set the `Class`/`Name count to use...` to `100`